Command/Validators.php

```diff
@@ -96,11 +96,23 @@ public static function validateFormat($format)
         return $format;
     }
 
+    /**
+     * Performs basic checks in entity name.
+     *
+     * @param string $entity
+     *
+     * @return string
+     *
+     * @throws \InvalidArgumentException
+     */
     public static function validateEntityName($entity)
     {
         if (false === strpos($entity, ':')) {
             throw new \InvalidArgumentException(sprintf('The entity name must contain a : ("%s" given, expecting something like AcmeBlogBundle:Blog/Post)', $entity));
```

Comment:
Yes, since it's checked by the regex. Done.
It was kept here because this check is so specific, thus I thought that maybe somebody trust this exception message.